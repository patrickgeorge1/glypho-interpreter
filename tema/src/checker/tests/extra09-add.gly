999GwwwI;O;z_}}}