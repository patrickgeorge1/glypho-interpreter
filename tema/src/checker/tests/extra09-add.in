19283574134626341586718045689873568987243564212345
-394587634986734259874162324713658744598567843
