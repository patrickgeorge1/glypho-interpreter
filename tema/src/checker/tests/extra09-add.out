19283179546991354852458171527548855328498965644502
